lib/ansible/modules/apt_repository.py

```diff
@@ -142,9 +142,9 @@
 
     - name: somerepo | apt source
       ansible.builtin.apt_repository:
-      repo: "deb [arch=amd64 signed-by=/etc/apt/trusted.gpg.d/somerepo.asc] https://download.example.com/linux/ubuntu {{ ansible_distribution_release }} stable"
-      state: present
-'''
+        repo: "deb [arch=amd64 signed-by=/etc/apt/trusted.gpg.d/somerepo.asc] https://download.example.com/linux/ubuntu {{ ansible_distribution_release }} stable"
```

Comment:
You can make the line shorter instead of adding the noqa, similar to https://github.com/ansible/ansible/pull/78919/files

Reply:
thank you. Did that.